guides/gpt5_prompting_guide_pt-br.md

```
@@ -0,0 +1,519 @@
+# GPT-5 Guia de Prompting
+
+# GPT-5 Guia de Prompting
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)